Makefile

```diff
@@ -5,18 +5,27 @@ default: help
 help:
 	@echo "make help              Show this help message"
 	@echo "make services          Run the services that Concourse and the pipeline requires"
+	@echo "make initdb            Restore DB dump on first run, can be also used to restore DB"
 	@echo "make sql               Connect to the cnx-db database with a psql shell"
 
-.PHONY: services
-services:
-ifneq ("$(wildcard ./20190805_dump.sql.gz)","")
-	docker-compose up -d
-else
+
+dump.sql.gz:
 	@echo Downloading db dump file first...
-	rsync -tv --progress -e 'ssh -T -c aes128-ctr -o Compression=no -x' backup1.cnx.org:/var/backups/db_dump/20190805_dump.sql.gz ./
+	rsync -tv --progress -e 'ssh -T -c aes128-ctr -o Compression=no -x' backup1.cnx.org:/var/backups/db_dump/20190805_dump.sql.gz ./dump.sql.gz
+	@echo Please run "make initdb" to restore dump
+
+
+.PHONY: services
+services: dump.sql.gz
```

Comment:
`initdb` should be the one dependent on `dump.sql.gz` right?

Reply:
makes probably more sense. Will change.